### DISCUSSION TITLE
Support set void function to the event listener

### DISCUSSION THREAD

**Discussion by junwchina** [jeffreylanters/react-unity-webgl#517] [Feature Requests]
Hi, could we make a little change to ```ReactUnityEventParameter``` to support void function?  sometimes, we might not want to return anything from listener.

use cases:
```
// Unity is started, we may do something on typescript side. 
// no return needed
const onStarted = () => {
    // do something here
}

addEventListener("OnStarted", onStarted);
```

The following changes is need to support this usecase:
```
// file: source/types/react-unity-event-parameters.ts
type ReactUnityEventParameter = string | number | undefined | void;
```

**Comment by jeffreylanters** [comment]
Hi! _Sorry for me late response, I've been away for a couple of days._ Thanks for your suggestion, I've added support for void functions in the latest release 9.4.3!
